hummingbot/strategy/arbitrage/arbitrage_config_map.py

```diff
@@ -47,13 +47,13 @@ def secondary_market_on_validated(value: str):
                   default="arbitrage"),
     "primary_market": ConfigVar(
         key="primary_market",
-        prompt="Enter your primary exchange name >>> ",
+        prompt="Enter your primary spot connector >>> ",
```

Comment:
Many users might not know what a sport connector. I think we can keep exchange name.

Reply:
Keeping this as "exchange" would bring up all connectors(including derivatives) which isn't ideal.
I think we should use as it is and make changes when we standardizes names as discussed during last planning meeting,

Reply:
You can make the search for `exchange` but not `derivative exchange` to get a list of spot exchanges. 
I agree we should wait for the standardised names before apply them all at the same time. To avoid confusing users, we better keep the terms as they are for now.